config/example copy/access levels.txt

```diff
@@ -0,0 +1,79 @@
+HOW TO CONVERT A MAP TO THE NEW (june 2008) ACCESS LEVEL SYSTEM
```

Comment:
This folder should not be part of this PR

Reply:
I... Don't even know what this is or how it ended up in my PR? I didn't touch anything that has anything to do with access. How do I remove it form the PR? click "delete file"? or will that make the github think i want this file deleted as part of the merge?

Reply:
Delete the example copy folder on your PC inside this branch, commit

Reply:
Okay, did that and committed. though now the "push" button changed to "pull origin". Do I need to do anything? Force push? (Sorry, I'm still new at this.)

Reply:
Try to pull origin, then it should let you push

Reply:
Okay, done! Thank you! Hope that fixes it!